### PR TITLE
maybe bad solution fix internal storage - add jni option

### PR DESCRIPTION
**Pull request by MatejMagat305**

### CODE REVIEW

**Comment by MatejMagat305** [comment]
this contains solution of  #365 and example which I test

File: raylib/platforms/rcore_android.c [hunk 1]
```diff
@@ -129,7 +129,7 @@ void android_main(struct android_app *app)
 }
 
 // NOTE: Add this to header (if apps really need it)
-struct android_app *GetAndroidApp(void)
+extern struct android_app *GetAndroidApp(void)
```
Comment (MatejMagat305) [inline raylib/platforms/rcore_android.c:132]: reason for this is in pure C we can make `#include "core_android.h"`, but in go would be pain ...

File: raylib/platforms/rcore_android.c [hunk 2]
```diff
@@ -134,6 +135,11 @@ struct android_app *GetAndroidApp(void)
     return platform.app;
 }
 
+extern struct ANativeActivity *GetANativeActivity(void)
```
Comment (MatejMagat305) [inline raylib/platforms/rcore_android.c:138]: reason for this is in pure C we can make #include "rcore_android.h" for get jni, but in golang would be pain ...

**Comment by gen2brain** [comment]
Fixed in https://github.com/gen2brain/raylib-go/commit/9b53342ac0617583005414aac357b1854cf58fae, thanks.

**Comment by MatejMagat305** [comment]
I thank, that is solve

**Pull request closed without merging**